FLEDGE.md

```diff
@@ -1246,7 +1310,104 @@ The following table summarizes which APIs get original and which get converted b
 | Private Aggregation `winning-bid` | Original value | Converted value |
 | Private Aggregation `highest-scoring-other-bid` | Original value | Converted value |
 
-#### 5.4 Losing Bidder Reporting
+#### 5.4 Reporting IDs
+
+Protected Audience provides several interest group fields that can be used to report details about a bid.  These fields needs to
+be jointly k-anonymous with the interest group owner, bidding script URL, and render URL to be provided to the reporting functions.
+Which of `selectableBuyerAndSellerReportingIds`, `buyerAndSellerReportingId`, `buyerReportingId`, and the interest group name gets
+passed to `reportWin()` and `reportResult()` is determined by the browser with the following logic:
+
+* If `selectedBuyerAndSellerReportingId` in bid:
+    * Then `selectedBuyerAndSellerReportingId`, `buyerAndSellerReportingId` (if present in interest group), and `buyerReportingId`
+      (if present in interest group) will all be available to reporting.
+* Otherwise, if `buyerAndSellerReportingId` defined in interest group: `buyerAndSellerReportingId` available to reporting.
+* Otherwise, if `buyerReportingId` defined in interest group: `buyerReportingId` available to reporting.
+* Otherwise, interest group `name` available to reporting.
+
+Here's a table representation of the above logic: (`selectableBuyerAndSellerReportingIds` abreviated to selectableBASRI,
+`buyerAndSellerReportingId` abreviated to BASRI, `buyerReportingId` abreviated to BRI)
+<table>
+  <thead>
+    <tr>
+      <th colspan=3>When interest group contains:</th>
+      <th colspan=2>then reports get:</th>
+    </tr>
+    <tr>
+      <th>selectableBASRI</th>
+      <th>BASRI</th>
+      <th>BRI</th>
+      <th>reportWin()</th>
+      <th>reportResult()</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>yes and in bid</td>
+      <td>optional</td>
+      <td>optional</td>
+      <td>selectableBASRI, BASRI, BRI</td>
+      <td>selectableBASRI, BASRI</td>
+    </tr>
+    <tr>
+      <td>no or not in bid</td>
+      <td>yes</td>
+      <td>optional</td>
```

Comment:
One thought - should this "optional" be "ignored", since the BRI is not used anywhere when the BASRI is provided?